src/edu/stanford/dlss/wowza/SulWowza.java

```diff
@@ -156,17 +156,23 @@ public void initHoneybadger()
         if(environment == null)
             environment = new SulEnvironment();
 
-        String apiKey = environment.getEnvironmentVariable(HONEYBADGER_KEY);
+        String apiKey = environment.getEnvironmentVariable(HONEYBADGER_API_KEY_ENV_VAR);
+        String honeybadgerEnv = environment.getEnvironmentVariable(HONEYBADGER_ENV_NAME_ENV_VAR);
```

Comment:
technically, these could be considered static constants.  But I don't really mind them as is.